core/services/wifi/frontend/index.html

```diff
@@ -0,0 +1,312 @@
+            <div class="cell-2">
+              <div class="d-flex flex-justify-center">
+                <i :class="signalStrengthClass(network)"></i>
+              </div>
+            </div>
+          </div>
+
+          <div  
+              v-if="connected_network"
+              class="dialog" 
+              data-role="dialog" 
+              id="disconnection_dialog">
+            <div class="dialog-title">{{ connected_network.ssid }}</div>
+            <div class="dialog-content">
+              <div
+                v-for="(value, name) in connected_network"
+                :key="name">
+                <span>{{name}}: {{ value }}</span><br />
+              </div>
+            </div>
+            <div class="dialog-actions">
+              <button class="button js-dialog-close" @click="disconnectFromNetwork">
+                Disconnect
+              </button>
+            </div>
+            <span class="button square closer js-dialog-close"></span>
+          </div>
+
+          <div class="dialog" data-role="dialog" id="connection_dialog">
+            <div class="dialog-title">{{ selected_network.ssid }}</div>
+            <div class="dialog-content">
+              <div
+                v-for="(value, name) in selected_network"
+                :key="name">
+                <span>{{name}}: {{ value }}</span><br />
+              </div>
+            </div>
+            <div class="dialog-actions">
+              <input
+                id="passwordInput"
+                type="password"
+                placeholder="Password"
+                data-role-checkbox="true"
+              />
+              <button class="button js-dialog-close" @click="connectToNetwork">
+                Connect
+              </button>
+            </div>
+            <span class="button square closer js-dialog-close"></span>
+          </div>
+
+        </div>
+      </div>
+    </div>
+  </body>
+
+  <script>
+    /*global Metro, axios, Vue*/
+
+    new Vue({
+      el: '#app',
+      data () {
+        return {
+          available_networks: [],
+          Metro: null,
+          connected_network: null,
+          selected_network: {},
+          alert_box: null,
+          wpa_state: 'starting',
+        }
+      },
+      mounted () {
+        this.API_URL = '/v1.0'
+        this.Metro = Metro
+        this.Metro.init()
+        setInterval(this.updateData, 5000)
+      },
+      computed: {
+        networksToShow () {
+          let showable_networks = this.available_networks.map((network) => {
+            if (network.ssid) return network
+            return {...network, ssid: '[HIDDEN_SSID]'}
+          })
+          if (showable_networks && showable_networks.length) {
+            if (this.connected_network) {
+              showable_networks = showable_networks.filter((network) => network.ssid != this.connected_network.ssid)
+            }
+            return showable_networks.sort((a,b) => a.signallevel > b.signallevel ? -1 : 1)
```

Comment:
can't you just do b.signal_level - a.signal_level ?

Reply:
No. [This is the intended compare function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description).

Reply:
Edit: didn't noticed the minus sign. Though for a moment it was a comparator.

Apparently this works, but I don't know why.

Reply:
Edit2: oh, ok. Since the subtraction will always led to positive or negative values, it works. Cool.

Reply:
Done.